ecs-init/Godeps/Godeps.json

```diff
@@ -15,8 +126,78 @@
 			"Rev": "c9ad0ce23f68428421adfc6ced9e6123f54788a5"
 		},
 		{
+			"ImportPath": "github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus",
```

Comment:
This looks like an old version of go-dockerclient; the current version no longer has its dependencies vendored.  We should not use this version of go-dockerclient with the vendored, rewritten imports.